src/Blocks/sources.jl

```diff
@@ -558,13 +554,13 @@ data input component.
 # Connectors:
   - `output`
 """
-@component function Input(; name, buffer)
+@component function SampledData(; name, buffer)
     pars = @parameters begin buffer = buffer end
     vars = []
     systems = @named begin output = RealOutput() end
     eqs = [
-        output.u ~ input(t, buffer),
+        output.u ~ get_sampled_data(t, buffer),
     ]
     return ODESystem(eqs, t, vars, pars; name, systems,
-                     defaults = [output.u => input(0.0, buffer)]) #TODO: get initial value from buffer
+                     defaults = [output.u => get_sampled_data(0.0, buffer)]) #TODO: get initial value from buffer
 end
```

Comment:
```suggestion
                     defaults = [output.u => get_sampled_data(0.0, buffer)]) #TODO: get initial value from buffer
end

@deprecate Input SampledData
```
Since a release was made with the `Input` name, we should avoid breaking it completely.